src/mtcp/mtcp_restart.c

```diff
@@ -1193,7 +1193,8 @@ remapExistingAreasToReservedArea(RestoreInfo *rinfo,
     } else if (mtcp_strcmp(area.name, "[vvar]") == 0) {
       rinfo->currentVvarStart = area.addr;
       rinfo->currentVvarEnd = area.endAddr;
```

Comment:
I think the next two lines are incorrect.  `mtcp_strcmp` returns 0 if the two strings are equal.  We should keep the original code.

Also, you're changing this for all CPUs.  That seems strange.  Other CPUs are not reporting a bug here.